### ISSUE TITLE
CTD late game

### ISSUE THREAD

**Issue by Tarzan737** [LoneGazebo/Community-Patch-DLL#9655]
_1. Mod Version (X.Y.Z). Current Version: 3.1.1_
NON-EUI Version

_2. DirectX Version_

_3. List of Other Mods_

Advanced air units V13
Giant death robot eraser V1
Global 2 Units per fort V1
Improvement airbase mod V12
Quick turns V10
Really advanced setup V15
Edit Unit names V1
Subs ignore borders V3

_4. Describe the Issue_

CTD one or few turns away from savegame

______________________________________________________________________________________
_5. Save and logs provided

[logandsave.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/10678321/logandsave.zip)

**Comment by axatin** [comment]
can you provide a zip-file of all those mods? thx

**Comment by Tarzan737** [comment]
Advanced air units V13 is to large to upload

[MODS.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/10678969/MODS.zip)

**Comment by axatin** [comment]
oh wait, we already had something similar. see https://github.com/LoneGazebo/Community-Patch-DLL/issues/9163. Have you done the fix described there?

**Comment by Tarzan737** [comment]
Yes! I did fix those lines in my version of air units mod, could u see that in logs again?

**Comment by axatin** [comment]
hadn't checked yet, just wanted to make sure it's not that error again. apparently it's not, I will look into it then

**Comment by axatin** [comment]
ok, it is the same again. these need to be changed:
![assult](https://user-images.githubusercontent.com/95587882/217648757-a9606849-b112-40dc-9a4f-e5ba4dfeb630.PNG)

**Comment by Tarzan737** [comment]
Yes but i should have changed all those lines in my version.. strange

**Comment by axatin** [comment]
hmm... please attach your version of the file MODS\Advanced Air Units (v 13)\Jets3\UnitP40Warhawk22.xml

**Comment by Tarzan737** [comment]
Yes! Thanks for taking time..
[UnitP40Warhawk22.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/10713821/UnitP40Warhawk22.zip)

**Comment by axatin** [comment]
oh, now I understand! this is a bomber, so the default unit AI should be "UNITAI_ATTACK_AIR" and not "UNITAI_ASSAULT_SEA", and that's what you have to change it to. (will be savegame-compatible btw)

go though the mod files, it should be:
UNITAI_ATTACK_AIR for bombers
UNITAI_DEFENSE_AIR for fighters

btw this mod seems not to be designed for VP, maybe you want to consider using one of the VP-compatible mods from the forum repository instead?

**Comment by Tarzan737** [comment]
Yes this seems to work.. thank u very much..

What mod do you think of? is it something similar to air units mod?

**Comment by axatin** [comment]
I know that there's this mod https://forums.civfanatics.com/threads/enhanced-air-warfare-4vp.656114/ which looks similar to the other one (but I haven't played either, so I can't say for sure if it is)
